### PR TITLE
revert!: "DBTP-958 Straighten up Postgres plans (#112)"

### PR DESCRIPTION
**Pull request by WillGibson**
So we can release non-breaking changes, then release the breaking changes on purpose.

### CODE REVIEW

**Review by gabelton — APPROVED** [review]

File: postgres/tests/unit.tftest.hcl [hunk 1]
```diff
@@ -312,21 +306,11 @@ run "aws_db_instance_unit_test_set_to_non_defaults" {
     error_message = "Should be: 20"
   }
 
-  assert {
-    condition     = aws_db_instance.default.iops == 3000
-    error_message = "Should be: 20"
-  }
-
```
Comment (WillGibson) [inline postgres/tests/unit.tftest.hcl:319]: Won't pass with the old config, will put it back when we revert the changes in this pull request.

File: postgres/tests/unit.tftest.hcl [hunk 2]
```diff
@@ -312,21 +306,11 @@ run "aws_db_instance_unit_test_set_to_non_defaults" {
...
   assert {
     condition     = aws_db_instance.default.instance_class == "db.t3.small"
     error_message = "Should be: db.t3.small"
   }
 
-  assert {
-    condition     = aws_db_instance.default.storage_type == "io2"
-    error_message = "Should be: io2"
-  }
-
```
Comment (WillGibson) [inline postgres/tests/unit.tftest.hcl:329]: Won't pass with the old config, will put it back when we revert the changes in this pull request.

File: postgres/tests/unit.tftest.hcl (hunk 1 above)
Comment (WillGibson) [inline postgres/tests/unit.tftest.hcl:319]: Actually added in a subsequent commit, but won't pass with the old config, will put it back when we revert the changes in this pull request.
